src/gui/qgsattributeform.cpp

```diff
@@ -1574,6 +1575,9 @@ void QgsAttributeForm::init()
 
         label->setBuddy( widgetInfo.widget );
 
+        if ( widgetInfo.widget->sizePolicy().verticalPolicy() != QSizePolicy::Fixed )
```

Comment:
is this enough?
should you also check for Maximum and Preferred?

Reply:
You are right thanks,  i changed it